example-get-started-experiments/README.md

```diff
@@ -13,17 +13,21 @@ Run these commands to force push it:
 
 ```shell
 cd build/example-get-started-experiments
-git remote add origin git remote add origin git@github.com:<slug>/example-get-started-experiments.git
+git remote add origin https://github.com/iterative/example-get-started-experiments.git
+git remote add fixture https://github.com/iterative/example-get-started-experiments-fixture.git
```

Comment:
@daavoo I would vote against this (using iterative directly here and sharing a command for the fixture update). We'll have a bunch of fixtures, we need also to start versioning them (like https://github.com/iterative/monorepo-fixture-v1 so that we don't break the existing tests a we migrate). This way it's easier to break things by copy pasting everything and running it w/o doing a check (e.g. replace iterative -> shcheklein to test things).

Reply:
Makes sense. I will update the instructions.
I think it would still be nice to have this run automatically on CI for the "demo" one